test/honeybadger/insights/absinthe_test.exs

```diff
@@ -0,0 +1,156 @@
+defmodule Honeybadger.Insights.AbsintheTest do
+  use Honeybadger.Case, async: false
```

Comment:
Is there something in here that prevents this from being async? Seems rather isolated because there isn't any config changes (that I see)

Reply:
Nope, oversight.

Reply:
I realized you were talking about more than just Absinthe so I enabled async for all the modules and things started failing. Since we are starting and stopping the test_helper running these tests in parallel throws an "application already running" error.